test/verify/check-system-info

```diff
@@ -559,35 +507,50 @@ revision\t: 2.3 (pvr 004e 1203)
         b.enter_page('/system/hwinfo')
         b.wait_in_text('#hwinfo-system-info-list .hwinfo-system-info-list-item:nth-of-type(2) .pf-c-description-list__group:nth-of-type(1) dd', "2x POWER9 (architected), altivec supported")
 
-        # Memory details should be shown for generic QEMU memory
-        # demidecode not available on certain images
-        if not m.ostree_image:
+        # Memory details should be shown from our mocked DMI information from systemd's test files.
+        # fedora-coreos does not have dmidecode
+        if m.image != "fedora-coreos":
```

Comment:
NB that `m.ostree_image` is currently exactly the same as `m.image == "fedora-coreos"`. We used to have 3 OSTree based OSes, but right now just one.

Why don't these three checks work on FCOS? isn't that the same code path as for the technology/rank? I originally really meant to *drop* the `if` completely, not rewriting the if condition.

Reply:
This is indeed true, and now the if is removed \o/